src/components/sbb-icon/sbb-icon.tsx

```diff
@@ -114,10 +114,12 @@ export class SbbIcon implements ComponentInterface {
         {this._svgIcon ? (
           <span class="sbb-icon-inner" innerHTML={this._svgIcon}></span>
         ) : (
-          <span class="sbb-icon-inner">
-            {/* To reserve space, we need an empty svg to apply dimension to. */}
-            <svg width="0" height="0"></svg>
-          </span>
+          this.name && (
```

Comment:
In my opinion I would keep the existing solution so that an empty name could have a meaning (to have a placeholder icon). So I'd suggest to fix it in the tab group story. What do you think?

Reply:
Ok, I can fix it from the CSS without changing the logic 👍